docs/content/manual/pre-release/spinning-disk/_index.md

```diff
@@ -0,0 +1,3 @@
+---
+title: Spinning Disk
```

Comment:
We can call it `resiliency` instead of spinning disk, which is too narrowly defined. This should also applies to the scenarios with low network or low CPU situations.

Reply:
I renamed it ``resiliency``.